operators/derive/to_dp.py

```diff
@@ -7,6 +9,21 @@
 from . import helpers
 
 
+def merge_array_fields(fieldnames):
+    def func(r):
+        # get rid of null fields
+        vals = filter(None, [r[name] for name in fieldnames])
+        # create a flat view over vals
+        vals = chain(*vals)
+        # remove duplicates
+        vals = set(vals)
+        # materialize as a list (don't have issue later with different iterable container types)
+        vals = list(vals)
```

Comment:
perhaps use `sorted` instead of `list` for consistent return values?

Reply:
Yes, great idea.